includes/Modules/Analytics.php

```diff
@@ -854,23 +854,41 @@ protected function create_data_request( Data_Request $data ) {
 					return true;
 				};
 			case 'GET:report':
-				$date_range = $data['dateRange'] ?: 'last-28-days';
-
-				$dimensions = array_map(
-					function ( $name ) {
-						$dimension = new Google_Service_AnalyticsReporting_Dimension();
-						$dimension->setName( $name );
+				$request_args = array( 'page' => $data['url'] );
```

Comment:
This works, but I think it'd be more clear to only set this key in an if clause, only if `! empty( $data['url'] )`.

Reply:
Updated.